src/exception.cpp

```diff
@@ -182,6 +181,7 @@ namespace fc
             } catch( ... ) {
                ss << "<- exception in to_detail_string.";
             }
+            ++itr;
```

Comment:
can we move ++itr to for loop line, and next line only has ss << "\n"; looks simpler.

Reply:
We get an extra `\n` but I'm ok with that.